source/develop/release-management/features/virt/hot-unplug-memory.html.md

```diff
@@ -47,8 +50,55 @@ any memory in order to guarantee that hot plugged memory is movable.
 
 ## How to hot unplug memory from a VM
 
+### WebAdmin
+
 The previously hot plugged memory appears in the form of memory devices in
 *Vm Devices* tab of the given VM.  You can remove any of the devices (assuming
 they have been hot plugged considering the constraints described above) and
 thus remove that amount of memory from the VM.  Once the memory device is
 successfully hot unplugged the device disappears from the device list.
+
+*Physical Memory Guaranteed* is decremented if necessary.
+
+If next run configuration exists and its memory value is same as size of memory
+of running VM before hot unplug, next run configuration is updated.
```

Comment:
Maybe worth to reformulate a bit, e.g.:

... and its memory values is the same as the memory size of the running VM before hot unplug, ... ?

Reply:
done